docs/pipeline.rst

```diff
@@ -92,10 +92,11 @@ to change according to the settings in the `.gitlab-ci.yml file in the www-confi
     are equivalent::
 
         $ bin/tag-release.sh --push -r mozilla
-        $ MOZ_GIT_REMOTE=mozilla bin/tag-release.sh --push
 
     And if you'd like to just tag and not push the tag anywhere, you may omit the ``--push``
-    parameter.
+    parameter ::
+
+        $ MOZ_GIT_REMOTE=mozilla bin/tag-release.sh --push
```

Comment:
I think we want to keep this command under the `bin/tag-release.sh` command so it's still clear they are equivalent, but in separate code blocks so you can copy one or the other.